odoo/tools/misc.py

```diff
@@ -1360,6 +1360,38 @@ def format_duration(value):
     return '%02d:%02d' % (hours, minutes)
 
 
+def format_duration_biggest_uom_str(timedelta):
```

Comment:
Looking a bit what exists I think you should really have a look at format_timedelta in babel.